### DISCUSSION TITLE
Problem Sonderzeichen (Kodierung)

### DISCUSSION THREAD

**Discussion by faulgurke** [e-netze/webgis-community#60] [Q&A]
Der Schalter für das Ausblenden des TOCs sieht bei einer Instanz nicht so aus, wie es aussehen sollte (Bild1), sondern wie Bild2.
Es handelt sich wahrscheinlich um ein Kodierungsproblem. Wo kann man das einstellen?
Ich bitte euch um zweckdienliche Hinweise.
Bild1:
![bild1](https://user-images.githubusercontent.com/100088983/233985981-32c904f5-298b-4be9-9d4c-8d144ab9e38f.png)
Bild2:
![bild2](https://user-images.githubusercontent.com/100088983/233985998-697f6c57-468e-4f50-beac-69c92060e61e.png)

**Comment by gis-eni** [comment]
Das "Zeichen" wird als CSS Style definiert:

``` css
.webgis-sidebar-toggle-button::after {
            content: "«"
}
```

Dieser Stil ist in deinem Layout definiert. Das "Layout" File findest du unter portal/viewerlayer/{portalseite-id oder default}/w1200.html

Was steht bei dir in diesem FIle für diesen Stil. Vielleicht muss du dieses File nur einmal UTF8 kodiert abspeichern. Oder du kopierst dir diese Datei von der Instanz, wo es geht.

**Reply by faulgurke** [reply]
Vielen Dank - in UTF-8 abspeichern hat geholfen.
